src/firefly/js/visualize/saga/CatalogWatcher.js

```diff
@@ -27,7 +25,37 @@ import CsysConverter from '../CsysConverter.js';
 import {COVERAGE_CREATED} from './CoverageWatcher.js';
 import {isImage,isHiPS} from '../WebPlot';
 
+
+
+
+
+/** type {TableWatcherDef} */
+export const catalogWatcherDef = {
+    id : 'CatalogWatcher',
+    watcher : watchCatalogs,
+    testTable : testTableForCatalogs,
+    actions: [TABLE_LOADED, TABLE_SELECT, TABLE_HIGHLIGHT, TABLE_UPDATE, TBL_RESULTS_ACTIVE,
+              TABLE_REMOVE, ImagePlotCntlr.PLOT_IMAGE, ImagePlotCntlr.PLOT_HIPS]
+};
+
+
+function testTableForCatalogs(table) {
+
+    const {tableMeta, tableData}= table;
+    if ( !tableMeta[MetaConst.CATALOG_OVERLAY_TYPE]) return false;
+    if (isLsstFootprintTable(table)) return false;
+    const columns= getCenterColumns(table);
+    if (!columns) return false;
+    if (!tableData.columns.find( isCName(columns.lonCol)) && !tableData.columns.find(isCName(columns.latCol))) {
+        return false;
+    }
```

Comment:
`getCenterColumns` is the same as `VOAnalyzer.findTableCenterColumns`.  I think this check is already done. util/__tests__/VOAnalyzer-test.js:76

Reply:
ok, I changed it.  When @cwang2016 added the VOAnalyzer I told her to just make `getCenterColumns` as pass through. Since used in several other places in the code, I added a `@deprecated` to `getCenterColumns`. I guess we can changed the others later.

Reply:
I changed my mind.  I am just going to remove `getCenterColumns` altogether and refactor all who use it.